pkg/util/ipset/OWNERS

```diff
@@ -0,0 +1,8 @@
+reviewers:
```

Comment:
This file was pushed in #57582. Can you remove here to keep this PR a little cleaner?

Reply:
Sure. I rebased to the HEAD. Thanks!